src/routes/question.js

```diff
@@ -4,36 +4,36 @@ const questionController = require('../controllers/question');
 
 const router = express.Router();
 
-router.get('/', questionController.getAllQuestions);
+// router.get('/', questionController.getAllQuestions);
```

Comment:
Please revert the changes to this file :)

Reply:
I did in the last commit